src/components/Feilresponser/feilresponser-nb.ts

```diff
@@ -0,0 +1,49 @@
+export default {
+    'generell.fant.ikke.behandling': 'Fant ikke behandling',
+    'generell.fant.ikke.aktør.id': 'Fant ikke aktør ID',
```

Comment:
```suggestion
    'generell.fant.ikke.aktør.id': 'Fant ikke AktørID',
```
Ref: https://navikt.github.io/pdl/#opplysningstyper-felles-metadata

Reply:
Evt. Fant ikke AktørIDen